stackcopy.py

```
@@ -723,25 +722,31 @@ def main():
                 for job in pending_copy_jobs:
-                    success = job['future'].result()
-                    if success:
-                        processed_count += 1
-                    else:
+                    try:
+                        success = job['future'].result()
+                        if success:
+                            processed_count += 1
+                        else:
+                            failed_count += 1
+                    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)